### ISSUE TITLE
Application compatibility with RHEL8

### ISSUE THREAD

**Issue by RG00334999** [StackStorm/st2#6196]
We are running on Stackstorm version 2,7 and OS is RHEL7.
We are upgrading the OS to RHEL8, so question is RHEL8 is compatible with Stackstorm version 2.7?

**Comment by RG00334999** [comment]
Any updates

**Comment by nzlosh** [comment]
You'd be best to install it yourself and determine if it works.  StackStorm v2.7 is 6 years old and no longer supported.

**Comment by mamercad** [comment]
I don't think so, I think [3.2.0](https://packagecloud.io/app/StackStorm/stable/search?q=st2web&filter=all&filter=all&dist=el%2F8) will be the lowest version you can do on RHEL8.

**Comment by RG00334999** [comment]
do we have official document?

**Comment by mamercad** [comment]
> do we have official document?

Have you taken a look at [this](https://docs.stackstorm.com/install/rhel8.html)?

**Comment by RG00334999** [comment]
thanks all for your response.

Can someone please help the compatibility chart .....our current version is 2.7 and postgrseql version is 9.2
we are planning on stackstorm upgrade but before that we have to upgrade the postgresql on high prioroty as this is our organization requirements...we need to know what all postgresql versions are compatible with Stackstorm 2.7 version

**Comment by nzlosh** [comment]
You might want to contact a StackStorm partner https://stackstorm.com/partners/ as it is very unlikely anyone from the community will able to assist you with your request to provide a compatibility matrix.
